Package.swift

```diff
@@ -53,7 +53,7 @@ let package = Package(
         Target(
             /** Generates Xcode projects */
             name: "Xcodeproj",
-            dependencies: ["PackageType"]),
+            dependencies: ["PackageType", "Build"]),
```

Comment:
As of now we only needed pkgconfig as a dependency to Xcodeproj, how about pkgconfig be extracted out into a different module which both Build and Xcodeproj can depend on?

Though if this kind of dependencies will increase in future then I guess we should make a "build-related" module and move pkgconfig there.

What do you prefer?